packages/twenty-server/src/engine/metadata-modules/object-metadata/object-metadata.service.ts

```diff
@@ -299,6 +313,7 @@ export class ObjectMetadataService extends TypeOrmQueryService<ObjectMetadataEnt
           ]
         : // No fields for remote objects.
           [],
+      isSoftDeletable: true,
```

Comment:
@charlesBochet do you think this should be configurable, derived from an argument?

Reply:
By default it's good!
I think all objects should be softDeletable long term, I don't see why some would not be

Reply:
@ijreilly can you merge main. it seems there is a conflict because isSoftDeletable has already been added in another pr

Reply:
test are failing but also I'm afraid the auto-merge will break the code